src/ctapipe/io/eventsource.py

```diff
@@ -134,23 +134,6 @@ def __new__(cls, input_url=Undefined, config=None, parent=None, **kwargs):
         return super().__new__(subcls)
 
     def __init__(self, input_url=None, config=None, parent=None, **kwargs):
-        """
```

Comment:
Why is the help string to `__init__` being removed?

Reply:
Because it had outdated information and `__init__` shouldn't be used at all for documentation in the numpy style guide. The class docstring should be used (and is):

https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring

Reply:
The class docstring is missing the constructor parameters mandated by the styleguide

> The constructor (`__init__`) **should** also be documented here, the [Parameters](https://numpydoc.readthedocs.io/en/latest/format.html#params) section of the docstring details the constructor’s parameters.
> An Attributes section, **located below** the [Parameters](https://numpydoc.readthedocs.io/en/latest/format.html#params) section,
(emphasis mine).

Reply:
I'd rather say that the section is wrongly labeled Attributes instead of parameters...

Reply:
fixed